addons/io_hubs_addon/debugger.py

```diff
@@ -976,7 +976,7 @@ class HubsSceneDebuggerRoomExportPrefs(bpy.types.PropertyGroup):
     avatar_to_viewport: bpy.props.BoolProperty(
         name='Spawn using viewport transform',
         description='Spawn the avatar in the current viewport camera position/rotation',
-        default=True, options=set())
+        default=False, options=set())
```

Comment:
Why is the 'Spawn the avatar in the current viewport camera position/rotation' property being changed to default to off?

Reply:
I found it annoying that it was enabled by default as most of the time I don't want that so I just disabled it as I was working on this. I can move it to another PR.